rolling/doc-build.yaml

```diff
@@ -0,0 +1,59 @@
+%YAML 1.1
+# ROS buildfarm doc-build file
+---
+build_environment_variables:
+  ROS_PYTHON_VERSION: 3
+canonical_base_url: http://docs.ros.org/en
+documentation_type: rosdoc2
+jenkins_job_priority: 87
```

Comment:
The priorities of the current doc jobs look like this:

* Indigo - 89
* Kinetic - 87
* Melodic - 85
* Noetic - 84

(obviously only the last two are active).

So that suggests that as we add in distributions, we make the older ones lower priority.  Given that we have Foxy, Galactic, and Rolling all active on the ROS 2 side, I think it would make sense for rolling to be 89, Galactic to be 85, and Foxy to be 87.  Does that make sense to you?